fgpyo/util/metric.py

```diff
@@ -210,7 +210,9 @@ def _parsers(cls) -> Dict[type, Callable[[str], Any]]:
         return {}
 
     @classmethod
-    def read(cls, path: Path, ignore_extra_fields: bool = True) -> Iterator[Any]:
+    def read(
+        cls, path: Path, ignore_extra_fields: bool = True, strip_whitespace: bool = True
```

Comment:
Should this default be `False`? Otherwise it will be a breaking change

```suggestion
        cls, path: Path, ignore_extra_fields: bool = True, strip_whitespace: bool = False
```